site/learn/Learn-Execution.md

```diff
@@ -47,7 +49,7 @@ In order to describe what happens when a query is executed, let's use an example
 }
 ```
 
-You can think of each field in a GraphQL query as a function or method of the previous type which returns the next type. In fact, this is exactly how GraphQL works. Each field on each type is backed by a function called the *resolver* which is provided by the GraphQL server developer. When a field is executed, the corresponding *resolver* is called to produce the next value.
+You can think of each field in a GraphQL query as a function or method of the previous type which returns the next type (or a subset of the next type, see **Providing extra data in a resolver** section). In fact, this is exactly how GraphQL works. Each field on each type is backed by a function called the *resolver* which is provided by the GraphQL server developer. When a field is executed, the corresponding *resolver* is called to produce the next value.
```

Comment:
I think this addition is confusing, what does it mean to return something that is a subset of the expected return type? We always expect a field that claims to return a User to return a User and not something smaller than that